Formula/o/odin.rb

```diff
@@ -20,7 +19,8 @@ class Odin < Formula
   end
 
   depends_on "glfw"
```

Comment:
there is no linkage against `glfw`, is  that desired?

Reply:
glfw is normally vendored in the compiler's libraries, glfw here is used to replace that vendored library with brew's distribution.

You can see it being symlinked line 71 and a test covers it working on line 171.

Reply:
got it, thanks!